workshop/exercises/html/leaflet.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="https://unpkg.com/leaflet@1.9.4/dist/leaflet.css" integrity="sha256-p4NxAoJBhIIN+hmNHrzRCf9tD/miZyoHS5obTRR9BMY=" crossorigin="" />
+        <!-- Make sure you put this AFTER Leaflet's CSS -->
+        <script src="https://unpkg.com/leaflet@1.9.4/dist/leaflet.js" integrity="sha256-20nQCchB9co0qIjJZRGuk2/Z9VM+kNiyxNV1lvTlZBo=" crossorigin=""></script>
+        <script src="https://unpkg.com/leaflet-imageoverlay-ogcapi@0.1.0/Leaflet.ImageOverlay.OGCAPI.js"></script>
+
+        <style>
+            #map {
+                height: 100vh;
+            }
+        </style>
+        <title>Leaflet.js</title>
+    </head>
+    <body>
+        <div id="map"></div>
+        <script>
+            var map = L.map("map", {
+                minZoom: 6,
+                maxZoom: 24,
+            }).setView([21, 71], 6);
+            L.tileLayer("https://tile.openstreetmap.org/{z}/{x}/{y}.png", {
+                attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>',
+            }).addTo(map);
+            L.imageOverlay
+                .ogcapi("http://localhost:5050", {
```

Comment:
On docker-compose you are [running pygeoapi on port 5000](https://github.com/geopython/diving-into-pygeoapi/blob/c62bbf60d68378a9659ce007e658ff70f0e01c98/workshop/exercises/docker-compose.yml#L43), but here you are using a different port. Like this, it won't work.

Reply:
ahh ! yeah I was actually running on 5050 ( because of MAC) :p . will fix it right away